### PR TITLE
Coding conventions for pattern matching

### PR DESCRIPTION
**Pull request by wkolowski**

### CODE REVIEW

**Comment by palmskog** [comment]
@wkolowski can you also please add a mention of the 2-space convention to Definitions with linebreaks? That's a different header.

**Comment by wkolowski** [comment]
@palmskog Something like it was already there ("two spaces indentation before definition body"), but I reworded it to emphasize it should only be done if a definition doesn't fit on a single line.

File: CODING_CONVENTIONS.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ Inductive lv_event_type : Type :=
 ### Definitions
 
 - C-style name
-- two spaces indentation before definition body
+- If the definition doesn't fit into a single line, indent its body with 2 spaces starting in the next line.
```
Comment (palmskog) [inline CODING_CONVENTIONS.md:123]:
```suggestion
- if a definition doesn't fit into a single line, indent its body with 2 spaces when starting on the next line.
```

**Review by palmskog — APPROVED** [review]
Fine by me, but one final suggestion.

**Pull request merged**